turborepo-tests/integration/tests/conflicting-flags.t

```diff
@@ -1,9 +1,9 @@
 Setup
   $ . ${TESTDIR}/../../helpers/setup_integration_test.sh
   $ ${TURBO} run build --daemon --no-daemon
-   ERROR  the argument '--\(no-\)daemon' cannot be used with '--no-daemon' (re)
+   ERROR  the argument '--daemon' cannot be used with '--no-daemon' (re)
```

Comment:
why did this ever need to be a `(re)` line?

Reply:
Looking at the history I think it was an accident.

Reply:
cool cool: removed it in de9af56f3413324d029d9bf20b992c7153c21066